ohmec.js

```diff
@@ -208,7 +208,22 @@ function onEachFeature(feature, layer) {
     if("labelScale" in feature.properties) {
       fontsize *= feature.properties.labelScale;
     }
-    let inner = '<text text-anchor="middle" x=' + widthd2 + ' y=' + heightd2;
+    let inner = '';
+    // if labelArc is defined, we first need to define the circular path that the text will traverse
+    // it is a circle with radius 'arc' that has a tangent at (50,h/2), either with the circle below
+    // and the text on the top (if arc > 0) or the circle above with the text on the bottom (arc < 0).
+    if("labelArc" in feature.properties) {
+      let arcval = feature.properties.labelArc;
+      let my   = heightd2 + 2*arcval;
+      let ar   = (arcval >= 0) ? arcval : -arcval;
+      let pos  = (arcval >= 0) ?  1 :   0;
+      let ar2n = arcval*2;
+      let ar2p = arcval*-2;
+      inner += '<path id="arcpath' + feature.id + '" stroke="none" fill="none" d="m 50,' + my;
```

Comment:
Found a bug in this. The SVG namespace is evidently global, so needed to uniquify the path ID with the feature ID.

Reply:
For the record, it only appears to be an issue when two SVGs are active at a time. But still this is the right fix.